AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -53,6 +53,18 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
   if ([segue.destinationViewController respondsToSelector:@selector(setAdblockPlus:)]) {
     [(id)segue.destinationViewController setAdblockPlus:self.adblockPlus];
   }
+
+#ifdef CONFIGURABLE_CUSTOM_FILTER_LIST_ENABLED
+  if ([segue.identifier isEqualToString:@"ConfigureFilterListsSegue"]) {
```

Comment:
Not really big deal. It is only present in dev builds, it does not go to production